checkstyle.xml

```diff
@@ -145,10 +145,9 @@
         <!-- Miscellaneous other checks.                   -->
         <!-- See http://checkstyle.sf.net/config_misc.html -->
         <module name="ArrayTypeStyle"/>
-        <module name="FinalParameters"/>
```

Comment:
no

Reply:
java 8 zet alle variabelen automatisch op final, dus dit is onnodig

Reply:
bron?

Reply:
Ja bron?

Reply:
https://docs.oracle.com/javase/specs/jls/se8/html/jls-4.html#jls-4.12.4

Reply:
https://stackoverflow.com/questions/20938095/difference-between-final-and-effectively-final